src/epcpm/pm_helper.py

```diff
@@ -138,7 +147,7 @@ def build_uuid_scale_factor_dict(
         if type_node is None:
             continue
 
-        if type_node.name == "sunssf":
+        if type_node.name == "staticmodbussf":
```

Comment:
This will break for regular SunSpec. Need to leave `sunssf` in.
```suggestion
        if type_node.name == "sunssf" || type_node.name == "staticmodbussf":
```
Unless there is a more clever way to do this?

Reply:
[773121a](https://github.com/epcpower/pm/pull/72/commits/773121a56640e4204d381751c10068cf322c7371)